spec/services/partner_fetch_requestable_items_service_spec.rb

```diff
@@ -21,7 +21,8 @@
       end
 
       it 'should return all active and visible items' do
-        expect(subject).to eq(organization.items.active.visible)
+        expected_items = organization.items.active.visible.map { |item| [item.name, item.id] }.sort
+        expect(subject).to eq(expected_items)
```

Comment:
I'm not the biggest fan of tests like this because all it's really doing is re-stating the original code. What we should really be doing is clearing the items, creating items with explicit names, and confirming the size and values of the resulting map. We can also create items that we know shouldn't be in the result (because they're inactive or invisible).

Reply:
> I'm not the biggest fan of tests like this because all it's really doing is re-stating the original code.

I'm in total agreement. I just wasn't sure if this was the place to rework it. That being said if I am understanding you correctly the test should be something like:

- build a partner with no items
- build an input list of items (visible/invisible, active/inactive)
- build an a expected output: above list but filter
- test something like expect { changing partner items from [] to input}. to change(service call).from([]).to(filtered list)